### PR TITLE
change code styles to dark

### PR DESCRIPTION
**Pull request by cnunciato**
fixes https://github.com/pulumi/docs/issues/8973

-------

help from Christian:

@susanev Threw this PR together to give you something to use as a guide for https://github.com/pulumi/docs/issues/8973.

[Hugo syntax highlighting](https://gohugo.io/content-management/syntax-highlighting) is powered by Chroma, and there's a built-in `hugo` subcommand that lets you generate a base set of styles using one of several built-in templates. For this PR, I did a couple of things to help point you in the right direction:

1. I ran this `hugo gen chromastyles` subcommand and passed it a dark theme that looked decent, just to give you something to use as a reference. Here's the command I ran: 

    ```bash
    hugo gen chromastyles --style=github-dark > themes/default/theme/src/scss/_chroma.scss
    ```

    The command writes to a file called `_chroma.scss`, which is then loaded by `main.scss` (replacing `_pygments.scss` --- IIRC Pygments was the tool Hugo used before Chroma).

2. I made a few tweaks in `_code.scss`, `_copybutton.scss`, and `_choosable.scss` to remove a few overrides and change a few others -- again just as a demonstration of how we override the base set of generated styles up to now. I _think_ these are the only places where we override these default/generated Hugo styles, but there might be a few other places here and there.

Feel free to click around in the Chrome styles gallery to find a dark-background that works for you. You should be able to use that `hugo gen` command to regenerate the baseline themes pretty easily and them override them as necessary.

Hope this helps!

### CODE REVIEW

File: themes/default/assets/css/marketing.css [hunk 1]
```diff
@@ -122,7 +122,7 @@ pre {
 		'Liberation Mono',
 		Menlo,
 		monospace; /* 1 */
-  font-size: 1em; /* 2 */
+  font-size: 16px;
```
Comment (susanev) [inline themes/default/assets/css/marketing.css:125]:
```suggestion
  font-size: 14px;
```

File: themes/default/theme/src/scss/_code.scss [hunk 2]
```diff
@@ -1,8 +1,9 @@
-@import "pygments";
+@import "chroma";
 @import "colors";
 
 code {
-    @apply bg-gray-100 rounded py-1 px-2 text-sm break-words;
+    @apply rounded py-1 px-2 text-sm break-words leading-relaxed;
+    font-size: 16px;
```
Comment (susanev) [inline themes/default/theme/src/scss/_code.scss:6]:
```suggestion
    font-size: 14px;
```

File: themes/default/theme/src/scss/_code.scss [hunk 3]
```diff
@@ -12,7 +13,7 @@ h4,
 h5,
 h6 {
     code {
-        font-size: 0.9em;
+        font-size: 16px;
```
Comment (susanev) [inline themes/default/theme/src/scss/_code.scss:16]:
```suggestion
        font-size: 14px;
```

File: themes/default/theme/src/scss/_code.scss [hunk 4]
```diff
@@ -22,10 +23,12 @@ pre,
 }
 
 pre {
-    @apply bg-gray-100 py-4 px-5 pr-8 overflow-x-auto leading-normal;
+    background: #1F2557;
+    @apply text-gray-400 py-4 px-5 pr-8 overflow-x-auto leading-normal;
 
     .code-mode-dark & {
-        @apply bg-gray-900 border-none shadow-md;
+        background: #1F2557;
```
Comment (susanev) [inline themes/default/theme/src/scss/_code.scss:30]:
```suggestion
        background: #1f2557;
```

File: themes/default/theme/src/scss/_code.scss [hunk 5]
```diff
@@ -173,7 +179,7 @@ div.highlight.line-numbers {
                     @apply p-0 border-none shadow-none overflow-visible;
 
                     code {
-                        @apply text-sm;
+                        font-size: 16px;
```
Comment (susanev) [inline themes/default/theme/src/scss/_code.scss:182]:
```suggestion
                        font-size: 14px;
```

**Review by anita-trimbur — APPROVED** [review]
lgtm!! 🚀

**Pull request merged**